client/src/components/CreatePoll/index.js

```diff
@@ -92,26 +92,33 @@ export default function CreatePoll ({ openCreatePoll, setOpenCreatePoll }) {
             setPreviewSourceTwo(reader.result);
         };
     };
-    
-    const handleSubmitPoll = (e) => {
+  
+    const handleSubmitPoll = async (e) => {
         e.preventDefault();
         if(!newPoll) return;
-        uploadImage(previewSourceOne, previewSourceTwo, newPoll);
-        setNewPoll(initialPoll);
+        
+        const newPollForUpload = {
+            imageOne: previewSourceOne, 
+            imageTwo: previewSourceTwo,
+            question: newPoll.question,
+            friendlist: newPoll.friendlist,
+            userId: newPoll.userId,
+        }
+
+        await uploadPoll(newPollForUpload);
+        setReload(!reload);
```

Comment:
I think i understand why you have a state called `reload` here, I'm guessing it's to force a re-render in other components. If you set a new poll up in the App.js state, you can probably avoid doing this. we can tackle this refactor in another PR :)